.github/workflows/ci.yml

```
@@ -26,16 +30,68 @@
+        run: |
+          make clean
+          make all -j
+
+      - name: Build and Package the tools (Windows)
+        if: matrix.os == 'windows-latest'
+        shell: msys2 {0}
+        run: |
+          make clean
+          make all -j
+          make windows-debug
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)